modules/Microsoft.ServiceFabric/clusters/.test/common/deploy.test.bicep

```diff
@@ -58,7 +58,7 @@ module testDeployment '../../deploy.bicep' = {
     azureActiveDirectory: {
       clientApplication: resourceGroupResources.outputs.managedIdentityPrincipalId
       clusterApplication: 'cf33fea8-b30f-424f-ab73-c48d99e0b222'
```

Comment:
Must have missed this before? Do we know what this ID is `cf33fea8-b30f-424f-ab73-c48d99e0b222`?

If it is one of our own, I guess a dependency is missing. If it isn't we should at least mention what it is.

Reply:
From what I could grasp from [here](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-tutorial-create-vnet-and-windows-cluster#create-azure-ad-applications-and-assign-users-to-roles) it looks like a missing dependency on our side and I'm afraid not that quick to set up. Probably something to add to the manual dependencies still to be created?

Reply:
Agreed. As the AAD is involved there's not much wiggle room here